issuetracker/gitlab/gitlab.go

```diff
@@ -33,14 +34,18 @@ func (it *IssueTracker) taskURLFrom(taskID string) string {
 	return taskID
 }
 
-// IssueAPIOrigin returns the URL for github's issue-fetching API
+// IssueAPIOrigin returns the URL for gitlab's issue-fetching API
 func (it *IssueTracker) issueAPIOrigin() string {
-	tokens := common.RemoveEmptyTokens(strings.Split(it.Origin, "/"))
-	if !strings.HasPrefix(tokens[0], "http:") && !strings.HasPrefix(tokens[0], "https:") {
-		tokens = append([]string{"https:"}, tokens...)
+	pattern := regexp.MustCompile(`^(?P<scheme>^(https?:))//gitlab.com/(?P<owner>[a-zA-Z0-9\-]+)/(?P<repo>[a-zA-Z0-9-_.]+)/?$`)
```

Comment:
gitlab can also be hosted on private server instances.

Any hostname should be valid here, not only `gitlab.com`, e.g. `gitlab.myorg.com`